### PR TITLE
Added execution and updated installation section to Readme

### PR DESCRIPTION
**Pull request by markovamaria**
Related-to: 
https://github.com/redis/redis-benchmarks-specification/issues/105
https://github.com/redis/redis-benchmarks-specification/issues/104

### CODE REVIEW

**Comment by markovamaria** [comment]
Hi @filipecosta90, could you please take a look? I've added execution section and slightly updated installation section.

By the way, I see 2 installation sections - in start of readme and below (which I updated). Should they be combined to 1?

**Review by slice4e — COMMENTED** [review]
Maria, you are right that there are 2 "Installation" section. I would agree that we should probably keep 1 of them. I would say the second one. 
Then , notice that there are 2 ways to execute this. We have the coordinator and we have the standalone runner. 
I think that you could add your execution section before the section: Testing out redis-benchmarks-spec-sc-coordinator
You could name it: Testing out the redis-benchmarks-spec-runner  ... 
And explain the difference.

**Comment by markovamaria** [comment]
> Maria, you are right that there are 2 "Installation" section. I would agree that we should probably keep 1 of them. I would say the second one. Then , notice that there are 2 ways to execute this. We have the coordinator and we have the standalone runner. I think that you could add your execution section before the section: Testing out redis-benchmarks-spec-sc-coordinator You could name it: Testing out the redis-benchmarks-spec-runner ... And explain the difference.

Hi @slice4e, I removed one extra Installation section. I moved Execution to Contribution part. As it has installation dependencies, I moved Installation section to Contribution part too. 

Honestly I slightly confused that Installation and Execution (testing out) is presented in Contribution part only.  What if I don't want to add new benchmarks, but want to see Redis pass rate on benchmarks-specification? O_o 
I would suggest to move it or duplicate before "Contributing guidelines". Please let me know what is preferable.

Also I added Navigation in top of page for easy usage.

**Comment by markovamaria** [comment]
Hi @slice4e , @filipecosta90 , could you please look at CI falures? Seems like there is some connection issue, running tox from Readme locally gives me the same problem. How may I fix it?

File: Readme.md [hunk 1]
```diff
@@ -5,16 +5,30 @@
...
+- [Directory layout](#directory-layout)
+  - [Specifications](#specifications)
+  - [Spec tool implementations](#spec-tool-implementations)
+- [Contributing guidelines](#contributing-guidelines)
+  - [Joining the performance initiative and adding a continuous benchmark platform](#joining-the-performance-initiative-and-adding-a-continuous-benchmark-platform)
+    - [Joining the performance initiative](#joining-the-performance-initiative)
+    - [Adding a continuous benchmark platform](#adding-a-continuous-benchmark-platform)
+      - [Installing package requirements](#installing-package-requirements)
+      - [Installing Redis benchmarks specification](#installing-redis-benchmarks-specification-implementations)
+      - [Testing out the redis-benchmarks-spec-runner](#testing-out-the-redis-benchmarks-spec-runner)
```
Comment (filipecosta90) [inline Readme.md:22]: rename it to> redis-benchmarks-spec-client-runner

**Review by filipecosta90 — DISMISSED** [review]

**Comment by filipecosta90** [comment]
Thank you @markovamaria . Approving!

**Pull request merged**